x/programs/rust/wasmlanche-sdk/src/program.rs

```diff
@@ -12,7 +12,7 @@ use thiserror::Error;
 /// Defer deserialization from bytes
 /// <div class="warning">It is possible that this type performs multiple allocations during deserialization. It should be used sparingly.</div>
 #[cfg_attr(feature = "debug", derive(Debug))]
-#[derive(BorshSerialize)]
+#[derive(BorshSerialize, PartialEq)]
```

Comment:
```suggestion
#[derive(BorshSerialize)]
```

Reply:
oops... Laziness is my friend

Reply:
Fixed by the use of a `matches!`. That can't enter my head!